src/helpers/shareHelper.js

```diff
@@ -3,6 +3,7 @@ import { Share } from 'react-native';
 import appJson from '../../app.json';
 import { QUERY_TYPES } from '../queries';
 
+import { mergeWebUrls } from './mergeWebUrls';
```

Comment:
the structure of the helper file names is mostly ...Helper.js, please apply here as well.
and what about placing it in `linkHelper` instead of creating a new file?

Reply:
Yes, it makes sense. I've made that change.
https://github.com/smart-village-solutions/smart-village-app-app/commit/0de11ba779b6755711897c4ca4939cada2092108